src/runtime/graph/graph_runtime.cc

```diff
@@ -500,6 +548,23 @@ void GraphRuntime::SetupOpExecs() {
   }
 }
 
+void GraphRuntime::SetupProfile() {
+  //setup data storage
+  op_start_time_ = std::vector<float>(this->num_nodes(),0);
```

Comment:
Since we are just appending values to these vectors, we don't need to initialize them with 0. Declaration is enough and we can simply push_back to them.